bliss/surveys/sdss.py

```diff
@@ -255,7 +254,7 @@ class PhotoFullCatalog(FullCatalog):
     """
 
     @classmethod
-    def from_file(cls, sdss_path, run, camcol, field, band):
+    def from_file(cls, sdss_path, run, camcol, field, sdss_obj):
```

Comment:
If you can get the shape of the image from the WCS itself then that sounds like a better idea!

Reply:
Do we need the shape of the image here? It'd be nice to decouple the catalog from the images.

Reply:
Oh I see, it's not see easy because `FullCatalog` requires `height` and `width`.

Reply:
No, the WCS object won't include height and width. Those are properties of an image, not a coordinate transformation.

Reply:
Let's merge this PR as is because it solves the problem it sets out to solve (i.e., #842). At some point it may make sense to come up with a new catalog class that stores locations in world coordinates rather than pixel coordinates.

Reply:
Right, but the WCS we would want to use are from the SDSS object, which does contain the array size (since it cones from a particular image). I guess that's a good argument for having the object itself as the parameter though, to avoid generic WCSs that don't have the shape info.

Reply:
I just reverted back to passing in the entire SDSS object and retrieving height/width by accessing the actual image plus added documentation.